src/js/components/Heading/StyledHeading.js

```diff
@@ -10,12 +10,19 @@ const sizeStyle = props => {
   const levelStyle = headingTheme.level[props.level];
   if (levelStyle) {
     const data = levelStyle[size];
-    const styles = [
-      css`
+    const styles =  
+    [
+      data ? css`
         font-size: ${data.size};
         line-height: ${data.height};
         max-width: ${data.maxWidth};
         font-weight: ${levelStyle.font.weight || headingTheme.weight};
+      ` :
+      css`
+        font-size: ${size};
```

Comment:
I like the direction here, I think it will be cleaner to put inline conditions so the styles that are not impacted by `data` won't be included on the wide condition of line #15.
so something along these lines:
```suggestion
        font-size: ${data ? data.size : size};
```

Reply:
Converted the big ternary expression into three little ones as suggested - is this what you had in mind @ShimiSun?